Helper/DocumentImporter.php

```diff
@@ -0,0 +1,199 @@
+<?php
```

Comment:
did not review the complete file. Cfr refactor instructions and I will review afterwards.